server/src/main/java/org/apache/druid/server/coordinator/DruidCoordinator.java

```diff
@@ -483,7 +483,7 @@ public void moveSegment(
         );
       }
 
-      final String toLoadQueueSegPath =
+      final String toLoadQueueSegPath = curator == null ? null :
```

Comment:
As far as I can tell, none of the arguments here depend on ZK. So, we can define the path even if we don't actually have a ZK.

Reply:
Fixed.